cdk/python/bedrock/bedrock/bedrock_stack.py

```diff
@@ -0,0 +1,261 @@
+from aws_cdk import (
+    Stack,
+    CfnParameter,
+    RemovalPolicy,
+    aws_kms as kms,
+    aws_iam as iam,
+    aws_s3 as s3,
+    aws_bedrock as bedrock
+
+)
+from constructs import Construct
+
+class BedrockStack(Stack):
+
+    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+       # Parameters
+        open_search_collection_arn = CfnParameter(self, 'OpenSearchCollectionARN',
```

Comment:
Using Parameters in CDK applications is not common, and can be considered an anti-pattern. Normally configuration is done using constructors for the construct and/or stack.

Reply:
Thanks Eric for your review and suggestion, please have another check on my new update and let me know if there is something more I need to modifiy. Thanks